__tests__/fixtures/upgrade/package-not-in-registry/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "yarn-test-git-repo": "https://github.com/juanca/e2e-test-repo#package-not-in-registry",
```

Comment:
These can be switched to `yarnpkg` once `package-not-in-registry` branch is created and [this PR is merged]().